File: tests/brain/fixtures/code/sample.py
"""Fixture module for code_chunking tests."""

import os

VERSION = "1.0"


def helper(x):
    return x + 1


class Widget:
    """A widget."""

    def __init__(self, name):
        self.name = name

    def render(self):
        return f"<{self.name}>"

File: tests/brain/fixtures/code/sample.rs
//! Fixture module for code_chunking tests.

const VERSION: &str = "1.0";

fn helper(x: i32) -> i32 {
    x + 1
}

struct Widget {
    name: String,
}

impl Widget {
    fn new(name: String) -> Self {
        Widget { name }
    }

    fn render(&self) -> String {
        format!("<{}>", self.name)
    }
}
